src/cfnlint/schemaManager.py

```diff
@@ -0,0 +1,96 @@
+"""
+Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: MIT-0
+"""
+
+import json
+import logging
+import os
+import platform
+
+import botocore.exceptions
+import boto3
+from cfnlint.template import Template
+
+LOGGER = logging.getLogger(__name__)
+
+
+class SchemaManager(object):
+    """Download schemas if necessary and validate modules"""
+    def __init__(
+            self, filename, template, regions):
+        self.filename = filename
+        self.regions = regions
+        self.template = Template(filename, template, regions)
```

Comment:
Would it be better if STS client and CFN client are initialized as instance variables of `SchemaManager`? So you don't have to initialize and passed them to method every single time.

Reply:
Great idea!